engine-src/Game/LambdaHack/Common/ClientOptions.hs

```diff
@@ -69,6 +71,7 @@ defClientOptions = ClientOptions
   , sallFontsScale = Nothing
   , sfonts = []
   , sfontsets = []
+  , sfullscreen = False
```

Comment:
Could you also, in a separate commit, add this to config file, just following closely how it's done for `sallFontsScale`?

Reply:
Yes, I was looking to add to config file but ran out of time.